source/unit/Fixture.ooc

```diff
@@ -170,9 +170,10 @@ Fixture: abstract class {
 }
 
 TestFailedException: class extends Exception {
+	description: String
```

Comment:
This variable will leak *if* the string passed is not a string literal, but if the user really needs a customized description, they will have to memory management it themselves.